server/helper_test.go

```diff
@@ -36,6 +37,7 @@ func setupTestHelper(t *testing.T) *testHelper {
 		// These mocks are replaced later, but serve the plugin during early initialization
 		msteamsAppClient:        &mocks.Client{},
 		disableCheckCredentials: true,
+		pluginStore:             &pluginstore.MockStore{},
```

Comment:
Do we need to mock the store?  We have a real server running at this point.

Reply:
I probably saw the mock for the client and since the pluginStore wasn't defined created a mock for it. Let me fix it.